ios/ReactNativeExceptionHandler.m

```diff
@@ -1,15 +1,37 @@
 #import "ReactNativeExceptionHandler.h"
-
+#import <mach-o/dyld.h>
 
 // CONSTANTS
 NSString * const RNUncaughtExceptionHandlerSignalExceptionName = @"RNUncaughtExceptionHandlerSignalExceptionName";
 NSString * const RNUncaughtExceptionHandlerSignalKey = @"RNUncaughtExceptionHandlerSignalKey";
 NSString * const RNUncaughtExceptionHandlerAddressesKey = @"RNUncaughtExceptionHandlerAddressesKey";
 volatile int32_t RNUncaughtExceptionCount = 0;
 const int32_t RNUncaughtExceptionMaximum = 10;
-const NSInteger RNUncaughtExceptionHandlerSkipAddressCount = 4;
-const NSInteger RNUncaughtExceptionHandlerReportAddressCount = 5;
+const NSInteger RNUncaughtExceptionHandlerSkipAddressCount = 0;
+const NSInteger RNUncaughtExceptionHandlerReportAddressCount = 15;
+
+
+void getSlide(long* pheader,long* pslide) {
+    for (uint32_t i = 0; i < _dyld_image_count(); i++) {
+        if (_dyld_get_image_header(i)->filetype == MH_EXECUTE) {
+            long slide = _dyld_get_image_vmaddr_slide(i);
+            const struct mach_header* header = _dyld_get_image_header(i);
+            if(pheader)*pheader=header;
+            if(pslide)*pslide=slide;
+
+            return;
+        }
+    }
+}
 
+long letMeCrash(){
```

Comment:
Also please remove these test functions as well.

Reply:
ok, I have opened a new PR. 
I had thought that only the commits before opening are visible, so I commit my own modify after open it, but it's not the truth. now I make a special branch for this,  hope it's ok this time~

Reply:
by the way , except those test functions,  getting and saving the `slide`  is useful for symbol address translation, would you like it to be merged?

Reply:
Yes. I want to merge everything except the test functions